scripts/wcarthage.sh

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env bash
-
-# BASED ON THIS COMMENT: https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323
-
-# wcarthage.sh
-# Usage example: ./wcarthage.sh update --platform ios
```

Comment:
We’ve been pointing developers to this script for a while now. These changes would most likely be going into a minor release, so for backwards compatibility, we should keep the script around. Let’s verify that it’s still possible to bootstrap an application target using this script and the accompanying .framework-based instructions, even as we change the installation instructions to recommend .xcframework going forward.